shared/repository/transaction/asset/signature.ts

```diff
@@ -10,6 +12,10 @@ class TransactionSignatureRepo implements IAssetRepository<IAssetSignature> {
         };
     }
 
+    byteSerialize(asset: IAssetSignature): Buffer {
```

Comment:
we need to refactor this design. serialization/deserialization should not be placed in Repository level